graphdatascience/session/gds_sessions.py

```diff
@@ -82,6 +84,23 @@ def __init__(self, api_credentials: AuraAPICredentials) -> None:
             client_secret=api_credentials.client_secret,
         )
 
+    def estimate(
+        self, node_count: int, relationship_count: int, algorithm_categories: Optional[List[AlgorithmCategory]] = None
+    ) -> SessionSizeByMemory:
+        if algorithm_categories is None:
+            algorithm_categories = []
```

Comment:
We need to double check, but I think when algorithm categories is empty, the result is always to use 8gb

Reply:
good point! is that an EstimationCLI problem?
Shouldnt we at least estimate the projection based on the counts?

Reply:
Not sure where this problem originates, either in console-api or the estimation cli.
Ideally the projection should be taken into account but as far as I recall that wasn't the case

Reply:
no the 400 was coming from trying to create an AuraDS in Aura API with a size of 4 or 2 or 1

Reply:
yes the Aura API does return only an available size, I think. Although that requires tenant info, which is not there